#! /bin/sh /usr/share/dpatch/dpatch-run
## 03_htmldoc_i.dpatch by Thibaut Paumard <paumard@users.sourceforge.net>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: remove yorz-specific code.
## DP: should be fixed upstream.

@DPATCH@

--- orig/doc/html/htmldoc.i	2007-11-21 15:45:50.000000000 +0100
+++ debian/doc/html/htmldoc.i	2007-11-19 23:42:19.000000000 +0100
@@ -233,10 +233,12 @@
        (strpart(tags(3,),-3:0)=="jpeg") |
        (strpart(tags(3,),-3:0)=="zlib");
      list = where(mask);
-     tags(3,list) = "yorz";
-     write, format = "writing index and doc pages for %s      \n", "yorz";
-     hdoc_funcindex, "yorz", tags(,list), to;
-     hdoc_funcdocs, "yorz", tags(,list), tags, to;
+     if (numberof(list)) {
+       tags(3,list) = "yorz";
+       write, format = "writing index and doc pages for %s      \n", "yorz";
+       hdoc_funcindex, "yorz", tags(,list), to;
+       hdoc_funcdocs, "yorz", tags(,list), tags, to;
+     }
       tags = tags(,sort(tags(3,)));
       rtndx = (tags(3,2:0)!=tags(3,1:-1))(cum)+1;
       rtname = array(string, rtndx(0));
